.github/workflows/foreman.yml

```
@@ -33,7 +34,15 @@ jobs:
           POSTGRES_PASSWORD: password
     strategy:
       fail-fast: false
-      matrix: ${{ fromJson(needs.setup_matrix.outputs.matrix) }}
+      matrix:
+        ruby: ${{ fromJson(needs.setup_matrix.outputs.matrix).ruby }}
+        node: ${{ fromJson(needs.setup_matrix.outputs.matrix).node }}
+        postgresql: ${{ fromJson(needs.setup_matrix.outputs.matrix).postgresql }}
+        task:
+          - 'test:units'
+          - 'test:functionals'
+          - 'test:graphql'
+          - 'jenkins:integration'
```

Hacking the planet: can avoid the Jenkins task and simply use a space for multiple tasks?
```suggestion
          - 'webpack:compile test:integration'
```

Alternatively we can create a `webpack:compile_if_needed` task and enhance `test:integration` (which I think comes from Rails itself) with it as a dependency.

---

yes, that'd work